tools/devtools/src/devtools/tabs/History.tsx

```diff
@@ -189,14 +203,16 @@ export function History({
             <div
               ref={scrollRef}
               style={{ width: '100%', overflowX: 'auto', minHeight: '46px' }}
+              className="history-slider-wrap"
+              data-length={events.length}
             >
               <Slider
                 dots
                 min={0}
                 marks={sliderMarks}
                 max={events.length - 1}
                 value={selectedEventIndexInfo.index}
-                step={1}
+                step={0}
```

Comment:
Is there a reason for changing the step to 0? I'm asking because the documentation states to use a number greater than 0 or null, so I'm curious. https://github.com/react-component/slider/?tab=readme-ov-file#common-api

Reply:
Oh... Thank you for pointing it out. I changed `step` in the past in order to fix the issue with the redundant trail which is already resolved above. I just forgot to change it into its original state. I'll make a change right away.